packages/about-you/api-client/src/getCategory.ts

```diff
@@ -1,10 +1,11 @@
 
 import { api } from './index';
 import { CategoryWith } from '@aboutyou/backbone/types/CategoryWith';
+import { BapiCategory } from '@aboutyou/backbone/types/BapiCategory';
 
-export type getCategorySearchParams = { ids: number[]; path: string; with: CategoryWith; depth: number} | any
+export type getCategorySearchParams = { ids: number[]; path: string; with: CategoryWith; depth: number}
 
-export default async function(options: getCategorySearchParams) {
+export default async function(options: getCategorySearchParams): Promise<BapiCategory[]> {
```

Comment:
you can set the default param values like that
```suggestion
export default async function(options: getCategorySearchParams = {}): Promise<BapiCategory[]> {
```

Reply:
But this won't work since {} does not fit type getCategorySearchParams. We have to do something like `export default async function(options: getCategorySearchParams = { ids: [], path: '', with: {}, depth: 0, includeHidden: false }):`